app/src/screens/dev/DevPrivateKeyScreen.tsx

```
@@ -0,0 +1,101 @@
+import Clipboard from '@react-native-clipboard/clipboard';
+import { useCallback, useEffect, useState } from "react";
+import { Button, Text, XStack, YStack } from "tamagui";
+import { unsafe_getPrivateKey } from "../../providers/authProvider";
+import { black, slate200, slate50, teal500, white } from "../../utils/colors";
+import { confirmTap } from "../../utils/haptic";
+
+interface DevPrivateKeyScreen {}
+
+const DevPrivateKeyScreen: React.FC<DevPrivateKeyScreen> = ({}) => {
```

Biome (1.9.4)

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)